src/Summoner/ProjectData.hs

```diff
@@ -92,14 +93,8 @@ showGhcVer Ghc802  = "8.0.2"
 showGhcVer Ghc822  = "8.2.2"
 showGhcVer Ghc843  = "8.4.3"
 
--- | Converts numeric dot-separated GHC version into 'GhcVer'.
 parseGhcVer :: Text -> Maybe GhcVer
-parseGhcVer "7.10.3" = Just Ghc7103
-parseGhcVer "8.0.1"  = Just Ghc801
-parseGhcVer "8.0.2"  = Just Ghc802
-parseGhcVer "8.2.2"  = Just Ghc822
-parseGhcVer "8.4.3"  = Just Ghc843
-parseGhcVer _        = Nothing
+parseGhcVer = inverseMap showGhcVer
```

Comment:
Great! I already like this function :sweat_drops: 
Could you also remove `supportedGhcVers` variable? It's `universe @GhcVer` from `relude`. Don't need to have extra variable.